packages/renderer/src/lib/docker-extension/PreferencesPageDockerExtensions.svelte

```diff
@@ -72,6 +72,7 @@ function deleteContribution(extensionName: string) {
         <input
           name="ociImage"
           id="ociImage"
+          aria-label="ociImage"
```

Comment:
I would choose: `OCI image name`

Reply:
Another note is that this input does not have a type. Since we are passing a text, we should add attribute `type="text"`. This is something ARIA spec. suggests (https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/textbox_role), but also we might consider the input of a `url` type which *warning* could introduce also a validation. So it accepts url form of the image name only. Now I would be very careful with `type="url"` as it probably also must accept local file (file:///path/to/image). Probably text suits better here.